bindings/Sofa/tests/Simulation/Node.py

```diff
@@ -138,3 +158,109 @@ def test_getItem(self):
+            self.assertFalse(hasattr(root,"obj2"))
+            self.assertEqual(len(root.objects), 0)
+
+        def test_moveChild(self):
+            root1 = Sofa.Core.Node("root1")
+            root2 = Sofa.Core.Node("root2")
+            child = Sofa.Core.Node("aChild")
+            root1.addChild(child)
+            self.assertEqual(len(root1.children), 1)
+            self.assertTrue(hasattr(root1,"aChild"))
+            root2.moveChild(child,root1)
+            self.assertEqual(len(root2.children), 1)
+            self.assertTrue(hasattr(root2,"aChild"))
+            self.assertEqual(len(root1.children), 0)
+            self.assertFalse(hasattr(root1,"aChild"))
+
+        def test_isInitialized(self):
+            root = Sofa.Core.Node("root")
+            root.init()
+            self.assertTrue(root.isInitialized())
+
+        def test_getAsACreateObjectParameter(self):
+            root = Sofa.Core.Node("root")
+            node = root.addChild("node")
+            node2 = node.addChild("node2")
+            self.assertEqual(node.getAsACreateObjectParameter(),"@/node")
+            self.assertEqual(node2.getAsACreateObjectParameter(), "@/node/node2")
+
+        def test_detachFromGraph(self):
+            root = Sofa.Core.Node("root")
+            node = root.addChild("node")
+            self.assertEqual(len(root.children),1)
+            node.detachFromGraph()
+            self.assertEqual(len(root.children),0)
+
+        def test_getMass(self):
+            root = Sofa.Core.Node("root")
+            root.addObject("MechanicalObject")
+            m = root.addObject("UniformMass", name="mass", vertexMass=0.1)
+            self.assertEqual(m,root.getMass())
+
+        def test_getForceField(self):
+            root = Sofa.Core.Node("root")
+            root.addObject("MechanicalObject")
+            ff = root.addObject('ConstantForceField', template="Vec3d", name="cff2")
+            self.assertEqual(ff, root.getForceField(0))
+
+        def test_getMechanicalState(self):
+            root = Sofa.Core.Node("root")
+            c = root.addObject("MechanicalObject")
+            self.assertEqual(c, root.getMechanicalState())
+
+        def test_getMechanicalMapping(self):
+            root = Sofa.Core.Node("root")
+            root.addObject("MechanicalObject", name="t1")
+            root.addObject("MechanicalObject", name="t2")
+            mm  = root.addObject("BarycentricMapping", input="@/t1", output="@/t2")
+            self.assertEqual(mm, root.getMechanicalMapping())
+
+        def test_sendMessage(self):
+            root = Sofa.Core.Node("rootNode")
+            node = root.addChild("node")
+            c = node.addObject(MyController(name="controller"))
+            root.sendMessage(root.getName(),"This is a test")
+            self.assertEqual(c.event_name, "This is a test")
+            self.assertEqual(c.userData, root.getName())
+            self.assertEqual(c.sender, root)
+
+
+def getTestsName():
```

Comment:
The getTestsName / runTests should be removed from all python test files as they're not necessary since #50